### PR TITLE
[VEN-2692]: Core pool Reward distributor on Zksync sepolia

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description
**NOTE**:  VIP-350 on BSC testnet must be executed before multisig vip-007

Resolves VEN

### CODE REVIEW

File: multisig/proposals/zksyncsepolia/vip-007/index.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    {
+      target: zksyncsepolia.VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [zksyncsepolia.XVS, XVS_REWARD_AMOUNT, XVS_STORE],
+    },
+    {
+      target: XVS_VAULT_PROXY,
+      signature: "setRewardAmountPerBlockOrSecond(address,uint256)",
+      params: [zksyncsepolia.XVS, REWARD_TOKEN_SPEED],
+    },
```
Comment (chechu) [inline multisig/proposals/zksyncsepolia/vip-007/index.ts:63]: We don't need to configure the XVS rewards in the XVS vault here. I would be focused only on the rewards in the RewardsDistributor contracts

File: multisig/simulations/zksyncsepolia/vip-007/index.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  let xvs: Contract;
+
+  describe("Pre-VIP behaviour", async () => {
+    before(async () => {
+      bridgeSigner = await initMainnetUser(BRIDGE, parseUnits("1", 18));
+
+      xvsVault = new ethers.Contract(XVS_VAULT_PROXY, XVS_VAULT_ABI, provider);
+      xvs = new ethers.Contract(XVS, XVS_ABI, provider);
+
+      await xvs.connect(bridgeSigner).mint(zksyncsepolia.VTREASURY, TREASURY_AMOUNT, { maxFeePerGas: "20000000000" });
```
Comment (chechu) [inline multisig/simulations/zksyncsepolia/vip-007/index.ts:49]: I would add a VIP in this PR instead, bridging the needed XVS from BSC testnet to zkSync. So, first we'll execute the VIP bridging the XVS to the treasury on zkSync and them we will execute the TX's configuring the rewards

Comment (GitGuru7) [reply multisig/simulations/zksyncsepolia/vip-007/index.ts:49]: [2793f09](https://github.com/VenusProtocol/vips/pull/348/commits/2793f09627521e9d595e592e60eae6ff73f9dfe5)

**Review by chechu — APPROVED** [review]

**Comment by GitGuru7** [comment]
BSC testnet tx to bridge XVS : https://testnet.bscscan.com/tx/0x424b8b7afde7e9032c113cc6702a6c972a7c672b71018440264bc70bda8f719e

**Comment by GitGuru7** [comment]
https://sepolia.explorer.zksync.io/tx/0x440b596a74d49a00469c644c7ac14a03784b4372cf2124d3b5b426d7c9b2dd49

**Pull request merged**